File: nodejs/a_simple_app.js
var testparams = {
  "name": "John Doe",
  "email": "jdoe@xyz.com",
  "zip": "27517"
}

function main(params) {
  console.log(params); 
  
  var out = {};
  for (var i in testparams) {
      out[i] = params[i];
  }
  return { out };
}

if (require.main === module)
  console.log(JSON.stringify(main(testparams), null, 2));

exports.main = main;

File: python/a-simple-app.py
import json

parameters = {
    "name":"John Doe",
    "email": "jdoe@xyz.com",
    "zip": "27517"
}

def main(params):
    out = {}
    for key in params:
        out[key] = params[key]        
    return {"out":out}

if vars().get("__name__","") == "__main__":
    print(json.dumps(main(parameters)))
